src/python/tests/test_configuration_manager.py

```diff
@@ -0,0 +1,653 @@
+                "and entry_name default") in str(error_exec)
+
+    async def test_set_category_item_value_entry(self):
+        """
+        Test updating of configuration.value for a specific key
+        :assert:
+            1. `default` and `value` in configuration.value are the same
+            2. `value` in configuration.value gets updated, while `default` does not
+        """
+        select_value_stmt = sa.select([_configuration_tbl.c.value]).select_from(
+            _configuration_tbl).where(_configuration_tbl.c.key == 'boolean')
+        await create_category(category_name='boolean', category_description='boolean type',
+                              category_value={
+                                  'info': {
+                                      'description': 'boolean type with default False',
+                                      'type': 'boolean',
+                                      'default': 'False'}})
+        try:
+            async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+                async with engine.acquire() as conn:
+                    async for result in conn.execute(select_value_stmt):
+                        assert result[0]['info']['value'] == 'False'
+                        assert result[0]['info']['default'] == 'False'
+        except Exception:
+            raise
+
+        await set_category_item_value_entry(category_name='boolean',
+                                            item_name='info', new_value_entry='True')
+        try:
+            async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+                async with engine.acquire() as conn:
+                    async for result in conn.execute(select_value_stmt):
+                        assert result[0]['info']['value'] == 'True'
+                        assert result[0]['info']['default'] == 'False'
+        except Exception:
+            raise
+
+    async def test_set_category_item_value_error(self):
+        """
+        Test updating of configuration.value when category_name does not exist
+        :assert:
+            Nothing happens / returned
+        :expect:
+            FOGL-522: When updating category item value, when the category_name
+            doesn't exist, an error should get returned
+        """
+        select_value_stmt = sa.select([_configuration_tbl.c.value]).select_from(
+            _configuration_tbl).where(_configuration_tbl.c.key == 'boolean')
+        await set_category_item_value_entry(category_name='boolean',
+                                            item_name='info', new_value_entry='True')
+        try:
+            async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+                async with engine.acquire() as conn:
+                    async for result in conn.execute(select_value_stmt):
+                        assert result is None
+        except Exception:
+            raise
+
+    async def test_get_category_item_value_entry(self):
```

Comment:
name should be `test_get_set_category_item_value_entry` as it verifies set too ( update )